packages/grid/_modules_/grid/hooks/features/density/densityState.ts

```diff
@@ -0,0 +1,15 @@
+import { DEFAULT_GRID_OPTIONS, Density } from '../../../models/gridOptions';
+
+export interface DensityState {
+  size: Density;
+  rowHeight: number;
```

Comment:
rowheight is already in options. Is it a duplicate?
we should only have one source of truth